lms/product/moodle/_plugin/course_copy.py

```diff
@@ -31,6 +31,55 @@ def find_matching_group_set_in_course(self, course, group_set_id):
             course, group_set_id
         )
 
+    def effective_document_id(
+        self,
+        log,
+        document_url,
+        document_course_id,
+        document_file_id,
+        course,
+        current_course_id,
+        store_document_mappings,
+        get_document_mappings,
+        not_found_error_code,
```

Comment:
In general I'm a bit dismissive of pylint's `too many arguments" but it have a point here.